src/pages/stacks-blockchain/transactions.md

```diff
@@ -0,0 +1,487 @@
+
+Here are samples for Clarity value contructions using this library:
+
+```js
+// construct boolean clarity values
+const t = trueCV();
+const f = falseCV();
+
+// construct optional clarity values
+const nothing = noneCV();
+const something = someCV(t);
+
+// construct a buffer clarity value from an existing Buffer
+const buffer = Buffer.from('foo');
+const bufCV = bufferCV(buffer);
+
+// construct signed and unsigned integer clarity values
+const i = intCV(-10);
+const u = uintCV(10);
+
+// construct principal clarity values
+const address = 'SP2JXKMSH007NPYAQHKJPQMAQYAD90NQGTVJVQ02B';
+const contractName = 'contract-name';
+const spCV = standardPrincipalCV(address);
+const cpCV = contractPrincipalCV(address, contractName);
+
+// construct response clarity values
+const errCV = responseErrorCV(trueCV());
+const okCV = responseOkCV(falseCV());
+
+// construct tuple clarity values
+const tupCV = tupleCV({
+  a: intCV(1),
+  b: trueCV(),
+  c: falseCV(),
+});
+
+// construct list clarity values
+const l = listCV([trueCV(), falseCV()]);
+```
+
+If you develop in Typescript, the type checker can help prevent you from creating wrongly-typed Clarity values. For example, the following code won't compile since in Clarity lists are homogeneous, meaning they can only contain values of a single type. It is important to include the type variable `BooleanCV` in this example, otherwise the typescript type checker won't know which type the list is of and won't enforce homogeneity.
+
+```js
+const l = listCV < BooleanCV > [trueCV(), intCV(1)];
+```
+
+### Setting post-conditions
+
+The Stacks Transactions JS library supports construction of post conditions.
+
+Here is an example of the earlier mentioned post condition ("account's STX balance should have decreased by no more than 1 STX"):
+
+```js
+const account = 'SP2ZD731ANQZT6J4K3F5N8A40ZXWXC1XFXHVVQFKE';
+const comparator = FungibleConditionCode.GreaterEqual;
+// assuming the STX balance before the transaction is 12346
+const amount = new BigNum(12345);
+
+const standardSTXPostCondition = makeStandardSTXPostCondition(
+  account,
+  comparator,
+  amount
+);
+
+const postConditions = [
+  makeStandardSTXPostCondition(
+    postConditionAddress,
+    postConditionCode,
+    postConditionAmount
+  )
+];
+
+const txOptions = {
+  ..., // other transaction options
+  postConditions: [standardSTXPostCondition]
+}
+
+const transaction = await makeContractCall(txOptions);
+```
+
+## Serialization
+
+A well-formed transaction construct is encoded in [Recursive Length Prefix ("RLP")](https://eth.wiki/en/fundamentals/rlp). RLP encoding results in a variable-sized byte array.
+
+In order to broadcast transactions to and between nodes on the network, RLP data is represented in hexadecial (also called the **raw format**).
+
+To support an API-friendly and human-readable representation, transactions are serialized into a JSON format.
```

Comment:
The transaction's are serialized into a hex string for transmission. Only the blockchain API server will convert it into JSON for API call responses.